lib/src/helpers.dart

```diff
@@ -148,3 +150,61 @@ num convertArea(num area,
 
   return (area / startFactor) * finalFactor;
 }
+
+/// Creates a {@link Point} {@link Feature} from a Position.
+///
+/// @name point
+/// @param {Array<number>} coordinates longitude, latitude position (each in decimal degrees)
+/// @param {Object} [properties={}] an Object of key-value pairs to add as properties
+/// @param {Object} [options={}] Optional Parameters
+/// @param {Array<number>} [options.bbox] Bounding Box Array [west, south, east, north] associated with the Feature
+/// @param {string|number} [options.id] Identifier associated with the Feature
+/// @returns {Feature<Point>} a Point feature
+/// @example
+/// var point = turf.point([-75.343, 39.984]);
+///
+/// //=point
+Feature<Point> point(Position? coordinates, Map<String, dynamic> properties, {Map<String, dynamic>? options}) {
```

Comment:
I don't think, we need this helper method, because we can just use the class constructors for the same:
```dart
var feature = Feature(...);
var point = Point(...);
```